src/bot.js

```diff
@@ -15,7 +15,7 @@ const jsExt = Constants.jsExt;
  * Creates the discord client with interface hooks
  * @returns {Client<boolean>}
  */
-function createDiscordClient() {
+export function createDiscordClient() {
```

Comment:
Why is this here? Is this a bugfix discovered by testing?

Reply:
The `test/application/bot.test.js` needs access to this function looking at the code so this should be fine